src/app/modules/item/pages/item-current-situation/item-current-situation.component.html

```diff
@@ -1,24 +1,52 @@
 <ng-container *ngIf="itemData">
   <div class="selection-center" *ngIf="!hideSelection">
-    <alg-selection (change)="onViewChanged($event)" [items]="viewItems" [selected]="viewSelectedIndex"></alg-selection>
+    <nav class="tab-rounded">
+      <a class="tab-rounded-item"
+         [ngClass]="{'active': historyTab.isActive, 'prev': chapterTab?.isActive}"
+         [routerLink]="'./progress/history'"
+         routerLinkActive #historyTab="routerLinkActive"
+      >
+        {{ logItemCaption }}
+      </a>
+
+      <a class="tab-rounded-item"
+         [ngClass]="{'active': chapterTab.isActive, 'prev': chapterUserProgressTab?.isActive}"
+         [routerLink]="'./progress/chapter'"
+         routerLinkActive #chapterTab="routerLinkActive"
+         *ngIf="!showChapterUserProgress"
+         i18n
+      >
+        Chapter view
+      </a>
+
+      <a class="tab-rounded-item"
+         [ngClass]="{'active': chapterUserProgressTab?.isActive}"
+         [routerLink]="'./progress/chapter-user-progress'"
+         routerLinkActive #chapterUserProgressTab="routerLinkActive"
+         *ngIf="showChapterUserProgress"
+         i18n
+      >
+        Chapter view
+      </a>
+    </nav>
   </div>
 
   <div>
```

Comment:
Is it normal that "your situation" is empty?
 ![Screenshot 2021-05-26 at 14 47 10](https://user-images.githubusercontent.com/1053150/119662135-52fe3b00-be31-11eb-9c40-72c9397e2bab.png)
(on http://dev.algorea.org/branch/feature/item-progress-tab-revamp/en/#/activities/by-id/126952315730161957;attempId=0;path=4702,4102,1980584647557587953,39530140456452546/details/progress/history)

Reply:
Fixed